src/adapter/src/coord/dataflows.rs

```diff
@@ -412,6 +412,14 @@ impl<'a> DataflowBuilder<'a, mz_repr::Timestamp> {
         Ok(dataflow)
     }
 
+    /// Determine the given source's monotonicity.
+    fn monotonic_source(&self, source: &Source) -> bool {
```

Comment:
It's unused right now but it's because once we find a way to store the id of the parent source we'd like to use `self.catalog` to look it up and find its monotonicity

Reply:
ty